### DISCUSSION TITLE
URL in href links

### DISCUSSION THREAD

**Discussion by optimvsjag** [orestbida/cookieconsent#662] [General]
Hi team, 
I've a problem when change the Privacy Policies or Terms and Conditions URLs in cookieconsent-config.js file. I think that if the URL haven't a wright structure the consent popup not charge.

By default we have:
`consentModal: {footer: "<a href=\"#link\">Privacy Policy</a>\n<a href=\"#link\">Terms and conditions</a>"},`

I change **#link\** by **https://mydomain.com/privacy/** or **/privacy/** and cookieconsent not run, no charge the popup.

What is the correct URL structure for full URL or internal link? Thanks a lot.

**Comment by orestbida** [comment]
Probably a malformed configuration.

You can try changing the links in the playground, *Edit translations*. Alternatively you can use backticks:
```js
consentModal: {
    footer: `
        <a href="https://mydomain.com/privacy/">Privacy Policy</a>
        <a href="#link">Terms and conditions</a>
    `
}

```

**Reply by optimvsjag** [reply]
Thanks @orestbida, so in the original code **<a href=\"#link\">** the \ char after the = is left out, I'm confused there and I am not JS expert.

In the preferencesModal we have the contact link too, should be the same structure link?

I change this links and not runs yet.

Can you take a look if i'm missing some step? I only use spanish language in the texts.

site: https://vip.wowate.com
cookie consent file: https://vip.wowate.com/cookieconsent-config.js

Thanks a lot!

**Reply by optimvsjag** [reply]
Already works fine! I was editing cookiconsent-config.js file. I used the "Edit translation" app to update all my texts and links and now works perfectly.
The JS links have this structure and I didn't write correctly.
`<a href=\"/cookies/\">Cookies policy</a>"`
